### DISCUSSION TITLE
Preprocessing of the data

### DISCUSSION THREAD

**Discussion by yanhong-zhao-ef** [google/lightweight_mmm#45] [Q&A] [answered]
I am wondering do people see an increase in performance changing how preprocessing of the data is done?
Currently I am following the tutorial by de-meaning all variables in this way:
`media_scaler = preprocessing.CustomScaler(divide_operation=jnp.mean)`
But what are some other ways one could preprocess the data? By standardising it?

**Comment by pabloduque0** [comment, marked as answer]
We do recommend to divide by the mean because standardizing will lead to having negative values which does not make much sense for media units. 

If your media distribution within each channel is a bit odd you can always try using the `median`. 

Anyhow the specific operation or scaler you use should not affect performance much, or at least not of the sampling time (if anything it could slightly help on the media optimization side but I dont expect this to be a bottleneck by any means).
